types/jscodeshift/src/core.d.ts

```diff
@@ -26,7 +26,7 @@ declare namespace core {
     }
 
     interface FileInfo {
-        /** The absolute path to the current file. */
+        /** The path to the current file, relative to `process.cwd()`. */
```

Comment:
```suggestion
        /** The path to the current file. */
```

Reply:
Traced the contents of this value up to 
https://sourcegraph.com/github.com/facebook/jscodeshift@be4a67a7d10e6ad1ffa3ed57d8d73623efec0256/-/blob/bin/jscodeshift.js?L140:10#tab=references
Looks like whether the path is relative or absolute depends on what gets passed into the runner

Reply:
Tried a repo case to confirm this
```js
// transform.js
module.exports = function(fileInfo, api, options) {
  console.log(fileInfo)
  return source;
};
```

`./node_modules/.bin/jscodeshift -t transform.js foo.js`
which printed `{ path: 'foo.js', ... }`

`./node_modules/.bin/jscodeshift -t transform.js /Users/brieb/gists/jscodeshift-path/foo.js`
which printed `{ path: '/Users/brieb/gists/jscodeshift-path/foo.js', ... }`

Reply:
Agree that the comment as-is is inaccurate, so we can at least remove the absolute part which is misleading